src/components/tutorial-card/types.ts

```diff
@@ -0,0 +1,38 @@
+import { ProductSlug } from 'types/products'
+
+export interface TutorialCardProps {
+  /**
+   * A short description of the tutorial.
+   */
+  description: string
+
+  /**
+   * A string representation the duration of the tutorial, such as "10mins".
+   */
+  duration: string
```

Comment:
So you see the duration formatting calculation happening higher in the tree? Or would it feel reasonable for the tutorial card to do this?

Reply:
Ah i assumed it'd happen higher in the tree! Definitely feels reasonable for tutorial card to do this.

I think I'd plan to change `duration: string` here to `durationMinutes: number`:
1. Does that sound approximately right?
2. Would you be okay if we defer this change to when I'm implementing the collection view (slash rendering tutorial cards with some real data?). Just thinking it'll be easier to validate that things are working as expected if I'm using it with "wired up" data.

Reply:
Ya totally, it's v minor 🙌

Reply:
Sweet, I'll defer it till the next PR 👍